### ISSUE TITLE
Does this still work?

### ISSUE THREAD

**Issue by pschfr** [unsplash/unsplash-source-js#13]
I'm trying to use this in one of my personal projects ([pschfr/start](https://github.com/pschfr/start)) and cannot seem to get a response out of it.

Here's all the code I am using:
`var photo = new UnsplashPhoto();`
`photo.fetch();`

**Comment by lukechesser** [comment]
There is a definite chance it does not anymore 😕 

I'll look into it.

**Comment by pschfr** [comment]
Did you find anything out?

**Comment by lukechesser** [comment]
@pschfr checked today and it works for me. How are you loading the script?

**Comment by pschfr** [comment]
Via the good ol' script tag. It works now, did anything on your end change?

**Comment by lukechesser** [comment]
@pschfr ah good to hear. I did not :|

If it was generating the right URL but an image wasn't loading, it might have been a temporary outage. It's super rare for Source, but a few weeks back there were some pretty weird things going on with SSL and DDoS attacks with some of our providers. We keep everything up to date on status.unsplash.com though.

Closing this for now. Let me know if you see another problem 👍
